packages/hooks/src/useWalletMigration.ts

```diff
@@ -12,11 +12,20 @@ const useWalletMigration = () => {
         onSuccess: () => invalidate('wallet_migration'),
     });
 
-    /** Fetch the wallet_migration API and refetch it every second if the status is in_progress */
+    /** Fetch the wallet_migration API and refetch it regularly if the status is in_progress */
+    const [retryCount, setRetryCount] = useState(0);
     const { data } = useQuery('wallet_migration', {
         payload: { wallet_migration: 'state' },
         options: {
-            refetchInterval: response => (response?.wallet_migration?.state === 'in_progress' ? 500 : false),
+            refetchInterval: response => {
+                if (response?.wallet_migration?.state === 'in_progress') {
+                    const interval = 200 * Math.pow(2, retryCount);
+                    setRetryCount(prev => prev + 1);
```

Comment:
👍👍

Reply:
Have changed 200 to 500 because in testing there were issues, presumably because BE got overwhelmed, receiving 3 same requests in 1 sec